docs/tasks.md

```diff
@@ -156,10 +156,11 @@ If this field is present, the step cannot specify `command`.
 When specified, a `script` gets invoked as if it were the contents of a file in
 the container. Any `args` are passed to the script file.
 
-Scripts should start with a
-[shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) line to declare what
-tool should be used to interpret the script. That tool must then also be
-available within the step's container.
+Scripts that do not start with a shebang
+[shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) line will use a default
+value of `#!/bin/sh`, although users can override this by starting their script
```

Comment:
small suggestion, feel free to ignore since this is a dockerism, but `#!/usr/bin/env sh` will account for situations where sh is in /usr/bin or /usr/sbin of the image instead of /bin

Reply:
Yeah, I considered that but decided to stick with the docker version, I suppose there might be images with sh but not env?

Reply:
Yeah, totally fair point.